### ISSUE TITLE
Replace interactive links in Computer Graphics chapter

### ISSUE THREAD

**Issue by JackMorganNZ** [uccser/cs-field-guide#5]
The old computer graphics chapter used the following script for loading links:

``` html
<script>function linkSend(hrefID, capture_id, linkTo, parms){hrefID.href=linkTo+"?info="+ encodeURI(document.getElementById(capture_id).innerHTML);if(parms != null) hrefID.href += encodeURI(parms);}</script>
```

This needs to be replaced for the new style of interactive linking.

The original ReST text can be found [here](https://docs.google.com/document/d/1KfWA6-7nNIwfx0lVHGhZqYUAzzTJc-uDnM0n3Jzh51U/edit) (this is only viewable by original authors).

**Comment by JackMorganNZ** [comment]
Currently both text and numbers are passed through for the arrow matrix interactive. Best if all the data is stored in the interactive, and the examples are triggered as a parameter.

**Comment by JackMorganNZ** [comment]
This should be closed by #372, #373, #374, and #375.
